applications/callflow/doc/cf_branch_property.md

```diff
@@ -0,0 +1,42 @@
+/*
+Section: Callflows
+Title: Branch on callee's document property value
+Language: en-US
+Version: 4.0
+*/
+
+The `branch_property` callflow enables you to branch on value of some field inside callee's document (device, user or account).
+
+### Mandatory fields
+
+**property** - name of the property we are looking for
+
+### Optional fields
+
+**action** - "branch" - the only action supported now
```

Comment:
@amgunn there's no specific questions, we're looking at the implementation of the ticket. i still don't get why not use the merged properties from `cf_endpoint` (what's the use case not addressed by `cf_endpoint`). as to this particular comment, there's nowhere in the code a reference to this property, so it should be removed.